.github/workflows/release.yaml

```diff
@@ -121,7 +121,8 @@ jobs:
     - name: Build the Docker image
       run: |
         RELEASE_TAG=$(git describe --tags --abbrev=0)
-        ./scripts/dev_scripts/build_docker.sh "$IMAGE_NAME" ${{ github.workspace }} "$RELEASE_TAG"
+        export RELEASE_TAG
```

Comment:
Could we export the value directly?
```
export RELEASE_TAG=$(git describe --tags --abbrev=0)
```

Reply:
> Could we export the value directly?
> 
> ```
> export RELEASE_TAG=$(git describe --tags --abbrev=0)
> ```

Shellcheck recommends to keep them separate.